test/run.bats

```diff
@@ -358,3 +358,17 @@ EOF
     done
 }
 
+@test 'resolves relative path to image' {
```

Comment:
Call it "relative path to image".

You might also add a comment with the bug number.